bundles/best_practices.rst

```diff
@@ -397,10 +397,14 @@ Translation Files
 -----------------
 
 If a bundle provides message translations, they must be defined in the XLIFF
-format; the domain should be named after the bundle name (``acme_blog``).
+format; the domain should be named after the bundle name (``AcmeBlog``).
 
 A bundle must not override existing messages from another bundle.
 
+The translation domain should match the translation file names. For example,
```

Comment:
I think we should rather use „must“ than „should“ as the example otherwise doesn’t work.

Reply:
Indeed